node/impl/full/gas.go

```diff
@@ -297,6 +297,47 @@ func gasEstimateGasLimit(
 
 	ret := res.MsgRct.GasUsed
 
+	transitionalMulti := 1.0
+	// Overestimate gas around the upgrade
+	if ts.Height() <= build.UpgradeFVM1Height && (build.UpgradeFVM1Height-ts.Height() <= 20) {
+		transitionalMulti = 2.0
+
+		func() {
+			st, err := smgr.ParentState(ts)
+			if err != nil {
+				return
+			}
+			act, err := st.GetActor(msg.To)
+			if err != nil {
+				return
+			}
+
+			if builtin.IsStorageMinerActor(act.Code) {
+				switch msgIn.Method {
+				case 5:
```

Comment:
`case miner.Methods.SubmitWindowedPoSt`? (and so on for others)

Reply:
This corresponds to what is reported in collected statistics so it is easier to verify if it stays as numbers, also as this is very temporary code there is little to no harm to having raw numbers.

If you feel strongly about it I can add comments with method names.

Reply:
I don't feel strongly about it, no.

if you want me to double check the values against the collected statistics, i can, but i trust you